examples/jupyter/demo002_diffusion.ipynb

```diff
@@ -82,12 +82,11 @@
     "        θ = [θ_range[i]]\n",
     "        A = computesymbols(diffusion, θ)\n",
     "        eigenvalues = [abs(val) for val in eigvals(A)]\n",
-    "        maxeigenvalues[i] = max(eigenvalues...)\n",
-    "        maxeigenvalue = max(maxeigenvalue, maxeigenvalues[i])\n",
+    "        maxeigenvalues[i] = maximum(eigenvalues)\n",
```

Comment:
I don't know if this is a performance issue (depends whether unpacking into many arguments is eliminated), but `maximum` operates over arrays.